src/main/scala/net/jcazevedo/moultingyaml/YamlPrinter.scala

```diff
@@ -0,0 +1,67 @@
+package net.jcazevedo.moultingyaml
+
+import org.yaml.snakeyaml.{DumperOptions, Yaml}
+
+/**
+  * @author Simone D'Avico (simonedavico@gmail.com)
```

Comment:
My IDE adds that template on top of every new file. I wouldn't like to repeat it for every entity, there would be more comments than code. Once is enough :)